bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ManagedItemProvider.java

```diff
@@ -36,7 +39,7 @@
  * @author Kai Kreuzer - improved return values
  * @author Alex Tugarev - added tags
  */
-public class ManagedItemProvider extends AbstractManagedProvider<Item, String, PersistedItem> implements ItemProvider {
+public class ManagedItemProvider extends AbstractManagedProvider<Item, String, PersistedItem>implements ItemProvider {
```

Comment:
Are you using Eclipse Mars instead of Neo or why does your auto formatter removing the space?